src/components/Images/Images.scss

```diff
@@ -3,14 +3,14 @@
 
 // WIP.
 
+/* Generally used as a wrapper around Image components. */
 figure {
   margin: 0;
 }
 
 .Image {
   display: block;
-
-  // These are consequential, may need to pull out.
+  max-width: 100%;
   width: 100%;
-  height: auto;
+  object-fit: cover;
```

Comment:
Non Blocking:

Hrm, do we need to support other `object-fit` properties? I was looking at the MDN for what these was and I could see us wanting some of the others.

Reply:
I think it should be overridable if the user wants to pass through a classname and apply custom css. The reason I had to add this was to make it backwards compatible with the previous image component. Without it almost all the images on the CSS were getting horizontally skewed on smaller screens.